components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/protocol/kafka/KAFKATask.java

```diff
@@ -0,0 +1,48 @@
+/*
+ *  Copyright (c) 2005-2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.carbon.inbound.endpoint.protocol.kafka;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.synapse.core.SynapseEnvironment;
+import org.wso2.carbon.inbound.endpoint.common.InboundTask;
+
+public class KAFKATask extends InboundTask {
+    private static final Log logger = LogFactory.getLog(KAFKATask.class
+            .getName());
+    private KAFKAPollingConsumer kafkaPollingConsumer;
+
+    public KAFKATask(KAFKAPollingConsumer kafkaPollingConsumer,long interval) {
+        logger.debug("Initializing.");
+        this.kafkaPollingConsumer = kafkaPollingConsumer;
+        this.interval=interval;
```

Comment:
Fixed the formatting issue in the PR - https://github.com/wso2/carbon-mediation/pull/341